### ISSUE TITLE
Customize the Installing tab

### ISSUE THREAD

**Issue by vaind** [dart-lang/pub-dev#2967]
It would be useful to be able to add custom instructions to the auto-generated text in the "Installing" tab, or to replace the contents completely.

Goal: provide instructions on how to install up pre-compiled native libraries (for a package making use of FFI)

**Comment by jonasfj** [comment]
Yes, we've discussed rendering `INSTALL.md` as the installation tab. I think something like this would be useful.

**Comment by jonasfj** [comment]
We also discussed `install.md` in https://github.com/dart-lang/pub-dev/issues/2125.

**Comment by jonasfj** [comment]
As briefly discussed in https://github.com/dart-lang/pub-dev/issues/3008 another option is to remove the "installation" instructions completely.

Arguments for removal is that if installation instructions are important to your package, it's probably best to show them at the top of your `README.md` as users are unlikely to see them otherwise.
We could probably look at analytics to see if anyone is opening the "installation" tab.

**Comment by jonbhanson** [comment]
Users of my package, [flutter_native_splash](https://pub.dev/packages/flutter_native_splash), have contacted me with confusion because my readme tells them to add a in `dev_dependencies` (with is correct), but the installing tab tells them to add in `dependencies` (which is incorrect).  Would be nice to get this fixed!

**Comment by sigurdm** [comment]
> Users of my package, [flutter_native_splash](https://pub.dev/packages/flutter_native_splash), have contacted me with confusion because my readme tells them to add a in dev_dependencies (with is correct), but the installing tab tells them to add in dependencies (which is incorrect). Would be nice to get this fixed!

I think we should find a way to do: https://github.com/dart-lang/pub-dev/issues/6400 that would at least solve this specifc issue.

Maybe we still need a custom install tab on top of that.
